package.json

```diff
@@ -164,6 +164,7 @@
     "html-loader": "0.5.5",
     "html-loader-jest": "0.2.1",
     "html-webpack-plugin": "3.2.0",
+    "husky": "5.2.0",
```

Comment:
Can we talk about this in a different PR, with the whole team? I'm personally not a fan.

Reply:
ok i have remove code. we will discuss in the call